added language description

definemethod opened the pull request:
This should fix #132.

yurrriq commented on docs/ABOUT.md:
```diff
@@ -0,0 +1,9 @@
+Common Lisp is the most prominent, (together with Scheme) offshoot of the long-running family of Lisp programming languages. 
+It's a multi-paradigm programming language that allows you to to choose the approach and paradigm 
```
`s/to to/to`

verdammelt commented:
Perhaps along with Scheme, Clojure should be mentioned. It is currently probably *the* most prominent Lisp.

definemethod commented:
I went ahead and removed the reference to Scheme altogether. Mentioning both Scheme and Clojure made the description a bit cluttered in my opinion. I also removed the extra "to".

verdammelt commented:
This sounds good. It gets my vote 👍 

I'll wait a little while and then merge unless @yurrriq has any more comments/issues.

yurrriq commented:
LGTM
